keyboards/lendunistus/rpneko65/rev1/readme.md

```diff
@@ -0,0 +1,25 @@
+# rpneko65
+
+![rpneko65](https://i.imgur.com/QUIUSqJh.png)
+
+*PCB rear side*
```

Comment:
If I remember correctly, this should put a single line break after the image, with the text immediately below like a caption. But I can't check because my Markdown editor has stopped working. :slightly_frowning_face: 

```suggestion
![rpneko65](https://i.imgur.com/QUIUSqJh.png)\
*PCB rear side*
```